plugins/modules/storage/glusterfs/gluster_peer.py

```diff
@@ -141,7 +141,7 @@ def call_peer_commands(self):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            force=dict(type='bool', required=False),
+            force=dict(type='bool', required=False, default=False),
```

Comment:
Force is only tested for truthy in https://github.com/ansible-collections/community.general/blob/f03ca818dae586945c5d6a0c26a031e157f18449/plugins/modules/storage/glusterfs/gluster_peer.py#L91, so `False` and `None` are treated the same way.